assignments/callbacks.js

```diff
@@ -24,32 +24,89 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
```

Comment:
I will be going over callbacks today in stand up to make sure everyone understands